contact.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+    <div class="container">
+      <header>
+          <nav class="top-nav">
+              <a href="#"><img class="logo" src="img/lambda-black.png" alt="Lambda logo"></a>
+              <a href="./index.html">Home</a>
+              <a href="./about.html">About</a>
+              <a href="#">Products</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+          </nav>
+        </header>
+        <section class="top-contact">
+          <img class="contact-top" src="img/banner.jpg" alt="contact top banner">
+        </section>
+        <section class="contact-form">
+            <form action="#">
+            
+                <label for="fname">Full Name</label>
+                <input type="text" id="name" name="name" placeholder="First Last">
+            
+                <label for="lname">Email</label>
+                <input type="text" id="email" name="email" placeholder="test@email.com">
+            
+                <label for="country">Country</label>
+                <select id="country" name="country">
+                <option value="usa">USA</option>
+                <option value="australia">Australia</option>
+                <option value="canada">Canada</option>
+                
+                </select>
+            
+                <label for="subject">How can we help?</label>
+                <textarea id="subject" name="subject" placeholder="Your message here" style="height:100px"></textarea>
+            
+                <input type="submit" value="Submit">
+            
+            </form>
+        </section>
```

Comment:
I would have added the footer on this page as well just to keep every thing looking the same